components/CardDropdownComponent.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from "react";
+import { styled } from "styled-components";
+import { useModalSetting } from "../hooks/useModalSetting";
+import CommonModalComponent from "./CommonModalComponent";
+
+type props = {
```

Comment:
타입은 파스칼케이스로 작성해주는게 컨벤션입니다!
```suggestion
type Props = {
```